src/utils/fxa.js

```diff
@@ -79,15 +82,13 @@ async function getProfileData (accessToken) {
 
 async function sendMetricsFlowPing (path) {
```

Comment:
@mansaj note, I was not able to test the `sendMetricsFlowPing` function – not sure it's being used?

Reply:
Not for v2, I'm still trying to figure out what it does

Reply:
I think it's a part of the analytics